docs/source/starter/installation.rst

```diff
@@ -136,7 +135,23 @@ and run integration tests:
     # Run integration tests
     $GEOIPS_PACKAGES_DIR/geoips/tests/integration_tests/base_test.sh
 
-6. Test output
+6. Capture working requirements.txt for base install
```

Comment:
What's this step for? Just for maintaining working package versions?

Reply:
Yes, oh, also, at some point it said "OPTIONAL".  I am not sure how that is not in main or in the version release version...

Reply:
It was a bad merge.  Fixed it, now the only changes in installation.rst are the Miniforge updates....